cluster/ubuntu/config-default.sh

```diff
@@ -24,7 +24,7 @@ export nodes=${nodes:-"vcap@10.10.103.250 vcap@10.10.103.162 vcap@10.10.103.223"
 role=${roles:-"ai i i"}
```

Comment:
please make this sentence clear like "must be the same order with the nodes",now it is ambiguous.